api/apps/api/src/modules/geo-features/geo-feature.geo.entity.ts

```diff
@@ -11,12 +12,29 @@ export const geoFeatureResource: BaseServiceResource = {
   moduleControllerPrefix: 'geo-features',
 };
 
+export enum SourceType {
+  user_imported = 'user_imported',
+  gbif = 'gbif',
+  iucn = 'iucn',
+  ecoregions = 'ecoregions',
+  intersection = 'intersection',
+}
+
 @Entity('features_data')
 export class GeoFeatureGeometry {
   @ApiProperty()
   @PrimaryColumn()
   id!: string;
 
+  @Column('geometry', { name: 'the_geom' })
+  theGeom?: Geometry;
+
+  @Column('jsonb')
+  properties?: Record<string, string | number>;
```

Comment:
Are we able to narrow the type?

Reply:
are you asking if we can further narrow it, or if the narrowing here is accurate?

if the latter, looking at the dbase IV spec, I think we may only be missing boolean. I advised on this specific typing because of the way we import features via ETL pipelines (where bools are imported as string), but it won't hurt to add boolean here if that's what you meant.

Reply:
@hotzevzl I meant to provide more detailed structure rather than generic "everything can be there"